mpas_analysis/obs/observational_datasets.xml

```diff
@@ -91,7 +94,7 @@
     <bibtex>
     </bibtex>
     <dataUrls>
-      (missing)
+      (ftp://podaac-ftp.jpl.nasa.gov/allData/aquarius/L4/IPRC/v4/7day/)
```

Comment:
@milenaveneziani, I was thinking that these URLs would be for the individual data files, not just a directory.  The idea is that the script will eventually use them to download the data.  But maybe the better plan is just to hard-code the URL into the scripts themselves.  Especially for cases like this where hundreds of files are needed, a list of files is impractical.

Reply:
oh, there are many 7-daily files in this case..
Also, we need to think of a best way for *us* to maintain these scripts i the future, since changes to data sets are likely to happen.

Reply:
Obviously, this is well beyond the time frame of this PR ;-)